### PR TITLE
Use patches from GitHub to patch PHP-CSS-Parser

### PR DESCRIPTION
**Pull request by pierlon**
## Summary

In https://github.com/google/web-stories-wp/pull/5082 @westonruter & @swissspidy discussed the idea of using patches from GitHub instead of providing ones locally. The advantage with this approach is that the remote patches are easier to update, and it would resolve the issue of other packages not being able to apply our local patches (see https://github.com/cweagans/composer-patches/issues/315).

While working on this I found out the remote patch from https://github.com/sabberworm/PHP-CSS-Parser/pull/193 was not being applied cleanly, so I had to rebase with the latest changes from the `master` branch for it to succeed.

## Checklist

- [ ] My pull request is addressing an open issue (please create one otherwise).
- [x] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines#tests).
- [x] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

**Review by westonruter — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -56,9 +56,9 @@
...
-        "Validate name-start code points for identifier <https://github.com/sabberworm/PHP-CSS-Parser/pull/185>": "patches/php-css-parser-pull-185.patch",
-        "Fix parsing CSS selectors which contain commas <https://github.com/westonruter/PHP-CSS-Parser/pull/1>": "patches/php-css-parser-commit-10a2501.patch"
+        "Add additional validation for size unit <https://github.com/sabberworm/PHP-CSS-Parser/pull/193>": "https://github.com/sabberworm/PHP-CSS-Parser/compare/3bc5ded67d77a52b81608cfc97f23b1bb0678e2f%5E...468da3441945e9c1bf402a3340b1d8326723f7d9.patch",
+        "Validate name-start code points for identifier <https://github.com/sabberworm/PHP-CSS-Parser/pull/185>": "https://github.com/sabberworm/PHP-CSS-Parser/compare/d42b64793f2edaffeb663c63e9de79069cdc0831%5E...113df5d55e94e21c6402021dfa959924941d4c29.patch",
+        "Fix parsing CSS selectors which contain commas <https://github.com/westonruter/PHP-CSS-Parser/pull/1>": "https://github.com/westonruter/PHP-CSS-Parser/compare/master...10a2501c119abafced3e4014aa3c0a3453a86f67.patch"
```
Comment (pierlon) [inline composer.json:61]: @westonruter I'm using `master` as the `head` for this patch because https://github.com/westonruter/PHP-CSS-Parser/pull/1 includes a merge commit that would include a lot of unnecessary changes not relevant to the changes in the PR.

**Comment by pierlon** [comment]
The `lint` Travis job is currently failing due to `localheinz/composer-normalize` dropping support for Composer v1. That should be fixed when https://github.com/ampproject/amp-wp/pull/5554 is merged via https://github.com/ampproject/amp-wp/pull/5554/commits/fc5ddf2066ae61645f3ebbed587d5842e5e76f8f.

**Review by swissspidy — APPROVED** [review]

**Comment by pierlon** [comment]
:warning: Rebasing with latest changes from `develop`.

**Comment by westonruter** [comment]
I switched to `2.0` branch, cherry-picked fc3514a and 3984655 and then ran `composer update` then squashed into 295048799f417bbd62c3c21e2ed93c9a7de5c411.

**Comment by johnwatkins0** [comment]
@pierlon Any suggestions for QA testing this?

**Comment by pierlon** [comment]
Running `composer install` shouldn't result in an error. But that's already been proven since the command in the CI jobs has not been failing. This can be moved to "QA Passed".
